src/PointwiseFunctions/AnalyticSolutions/GrMhd/SmoothFlow.hpp

```diff
@@ -41,6 +41,20 @@ class SmoothFlow : virtual public MarkAsAnalyticSolution,
   using smooth_flow = RelativisticEuler::Solutions::SmoothFlow<3>;
```

Comment:
maybe as a separate PR after this one the Rel Euler solutions can be endowed with this capability? GRMHD can also use all the analytic solutions of the rel euler system. The solutions just set the magnetic field to zero. I think the same is true for the analytic data

Reply:
Yeah, agreed -- I've just been working on making it work with analytic data. There's a couple of extra tweaks needed for that, but it's all pretty easy.
I'll submit the next PR that makes those changes to all of the relevant data and solutions.